pkg/driver/node_linux.go

```diff
@@ -49,7 +52,15 @@ func (d *nodeService) findDevicePath(devicePath, volumeID string) (string, error
 	// /dev/disk/by-id/nvme-Amazon_Elastic_Block_Store_vol0fab1d5e3f72a5e23
 	nvmeName := "nvme-Amazon_Elastic_Block_Store_" + strings.Replace(volumeID, "-", "", -1)
 
-	return findNvmeVolume(nvmeName)
+	nvmeDevicePath, err := findNvmeVolume(nvmeName)
+	if err != nil {
+		//Return the nvmeDevicePath just for logging purpose
```

Comment:
the callers never read the returned string though.

Reply:
u r right... It is logging devicePath, will return empty string then